docs/core-concepts/versioning.md

```diff
@@ -1,6 +1,201 @@
+
+The next time .NET Core ships, it will (one assumes) add those new APIs such that it now supports that higher .NET Standard version. It can also add additional APIs at the same time, which could create a new .NET Standard version.
+
+.NET Core ships a Major Version
+-------------------------------
+
+When .NET Core ships a 2.0 version, the .NET Standard will be updated (assuming new APIs were added), but will stay with is 1.x versioning scheme. 
+
+We intend to move to a 2.0 version of .NET Standard only for very significant changes. We have defined a non-exhaustive set of cases where we would move to `netstandard2.0`, none of which we intend to do:
+
+- Drop APIs.
+- Add a major new feature that is potentially breaking for existing runtimes (e.g. array slices).
+- Update the CLR metadata/assembly format.
+
+Versioning in Practice
+======================
+
+There are commits and PRs on .NET Core repos on GitHub on a daily basis, resulting in new builds of many libraries. We don't want to create a new public version of .NET Core for every single change. Instead, we'll take changes for some loosely-defined period of time (e.g. weeks or months) before making a new public stable .NET Core version.
+
+A new version of .NET Core could mean several things:
+
+- New versions of packages and meta-packages.
+- New versions of various target frameworks.
+- New version of the .NET Core distribution.
+
+Shipping Bug Fixes (AKA "patch release")
+----------------------------------------
+
+After shipping a .NET Core v1.0.0 stable version, patch-level changes (no new APIs) are be made to .NET Core libraries to fix bugs and improve performance and reliability. The various meta-packages are updated to reference the updated .NET Core library packages. The meta-packages are versioned as patch updates (x.y.z).
+
+You can see patch updates demonstrated in the project.json examples below.
+
+```
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.1"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {}
+  }
+}
+```
+
+Target frameworks would not be updated. A new .NET Core distribution would be released with a matching version number to the Microsoft.NETCore.App meta-package.
+
+Shipping new APIs (AKA "minor release")
+---------------------------------------
+
+After shipping a .NET Core v1.0.0 stable version, new APIs are added to .NET Core libraries to enable new scenarios. The various meta-packages are updated to reference the updated .NET Core library packages. The meta-packages are versioned as patch updates (x.y) to match the higher target framework version.
+
+You can see minor updates demonstrated in the project.json examples below.
+
+```
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.1.0"
+  },
+  "frameworks": {
+    "netcoreapp1.1": {}
+  }
+}
+```
+
+The various target frameworks would be updated to describe the new APIs. A new .NET Core distribution would be released with a matching version number to the Microsoft.NETCore.App meta-package.
```

Comment:
Is there a place where we would say how people would update their stuff to make use of the new TFMs? I'm not sure this is the document for it but...or do we think it is self-explanatory?